app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<body>
+<main>
+  <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">Media Spotlight</span>
+      <% if @work %>
+      <%= link_to @work.title, work_path(@work.id), class: "spotlight__link-to" %> by <%= @work.creator %>
+        <%else %>
+      <%= @spotlight_sample %>
+        <% end %>
+    </h2>
+    <p class="spotlight__description">
+      <%= @work.votes.count%> votes - <%= @work.description%>
+    </p>
+  </section>
+
+  <hr class="root__hr" />
+
+  <section class="top-ten__container">
+    <section class="top-ten__list-container">
+      <h2 class="top-ten__header">
+        Top Movies
+      </h2>
+      <ul class="list-group top-ten__list">
+        <% @movies.each do |work| %>
```

Comment:
Note that you have 3 blocks of code that are very similar. Consider how you can use a partial view to DRY up this view code.